cli/internal/cmd/verifier.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/edgelesssys/constellation/v2/internal/constants"
+)
+
+type RekorVerifier interface {
```

Comment:
Can we unexport this interface, since it is currently only used inside the `cmd` package?